lib/diffcrypt/rails/encrypted_configuration.rb

```diff
@@ -68,6 +68,14 @@ def key
         read_env_key || read_key_file || handle_missing_key
       end
 
+      # It's required since the commit https://github.com/rails/rails/commit/1740b1f2cb8104435b6041ec6bfaabe58a6d74e6
+      # Returns truthy if #key is truthy. Returns falsy otherwise. Unlike #key,
+      # does not raise MissingKeyError when +raise_if_missing_key+ is true.
+      # @return [Boolean]
+      def key?
+        read_env_key || read_key_file
```

Comment:
This won't return a boolean, it'll return the value of the key or `nil` if not found.

I know the implementation is "truthy", but it would be better to match the real expected type inferred by `?` suffix.

```suggestion
        !!(read_env_key || read_key_file)
```

Reply:
Agree.